src/features/donations/components/ShareOptions.jsx

```diff
@@ -119,7 +120,7 @@ const ShareOptions = (props) => {
         className={styles.shareIcon}
         onMouseOver={() => setCurrentHover(4)}
         onClick={() =>
-          shareClicked(`https://twitter.com/intent/tweet?text=${textToShare}`)
+          shareClicked(`https://twitter.com/intent/tweet?hashtags=StopTalkingStartPlanting,TrillionTrees&via=PftP_int&url=${linkToShare}&text=${textToShare}`)
```

Comment:
Maybe use `donate:textToShare` here? https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/92fc7b0e60e32a7fbefdac4e7ba6b231740dbee2/public/static/locales/en/donate.json#L64